ui/src/dev-server.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+const express = require('express');
+const path = require('path');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+const webpackHotMiddleware = require('webpack-hot-middleware');
+const webpackConfig = require('../__config__/webpack.config.dev')({
+    browser: true,
```

Comment:
No, the compiler that is generated from the `webpackConfig` is only used for passing the browser bundle into `webpack-dev-middleware` and `webpack-hot-middleware`. The server bundle is loaded directly [when a request is received](https://github.com/guardian/frontend/pull/17809/files#diff-2ae5964175b603d2c233424b98b85891R27) (built by the serve bundle watch task), rather than compiled programmatically.